base/file.jl

```diff
@@ -572,14 +572,41 @@ struct uv_dirent_t
     typ::Cint
 end
 
+struct ReadDirIter
```

Comment:
This seems unnecessarily abbreviated. How about `ReadDirIterator`?

Reply:
Sure